documentation/deployment.md

```diff
@@ -217,6 +217,41 @@ IP.2 = 192.168.69.14
 IP.3 = 127.0.0.1
 {% endhighlight %}
 
+### Configuration environment variables
```

Comment:
should be ##

and ### for the followings

Reply:
Fixed